<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabItemForm</class>
 <widget class="QWidget" name="TabItemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
font-family: &quot;Microsoft YaHei&quot;;
color: #535353;</string>
  </property>
  <widget class="PreviewWidget" name="previewWidget" native="true">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>19</y>
     <width>145</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 170, 0)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>242</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/res/img/play-circle.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>189</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>设备名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="deviceNameText"/>
    </item>
    <item>
     <widget class="QPushButton" name="updateDeviceNameBtn">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>更新</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>125</y>
     <width>181</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="videoChk">
        <property name="text">
         <string>视频同步</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="audioChk">
        <property name="text">
         <string>声音同步</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QCheckBox" name="controlChk">
        <property name="text">
         <string>允许控制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="powerOffOnCloseChk">
        <property name="text">
         <string>自动熄屏</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>视频编码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="videoCodecList">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>音频编码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="audioCodecList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>54</y>
     <width>28</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S  N:</string>
   </property>
  </widget>
  <widget class="QLabel" name="snLabel">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>54</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="productLabel">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>77</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>77</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>产品：</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelLabel">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>100</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>型号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="borderLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>15</y>
     <width>155</width>
     <height>270</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/res/img/vphone.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loadLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/img/android.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>previewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
